.github/workflows/ci.yml

```diff
@@ -16,11 +16,11 @@ jobs:
       fail-fast: false
       matrix:
         os: [ubuntu, macos]
-        python-version: ['3.7', '3.8', '3.9', '3.10']
-        # test 3.11-dev and pypy on ubuntu only to speed up CI, no reason why macos X pypy should fail separately
+        python-version: ['3.7', '3.8', '3.9', '3.10', '3.11']
+        # test 3.12-dev and pypy on ubuntu only to speed up CI, no reason why macos X pypy should fail separately
         include:
           - os: 'ubuntu'
-            python-version: '3.11-dev'
+            python-version: '3.12-dev'
```

Comment:
Is that enough to add the trove classifier `Programming Language :: Python :: 3.12` already?